app/Http/Controllers/Api/VenueController.php

```diff
@@ -254,4 +255,17 @@ public function get_admin_venues(Authenticatable $user)
         $venues = Venue::where('user_id', '=', $user->id)->get();
         return VenueResource::collection($venues);
     }
+
+    public function suggest_shortest_path(Request $request)
+    {
+        $start = $request->query('start_coords');
+        $end = $request->query('end_coords');
+        $attributes = $request->query('attributes');
+
+        $suggester = new RouteSuggester($attributes, $start, $end);
+        $venues = $suggester->suggest();
+
+        if ($venues) return VenueResource::collection($venues);
```

Comment:
here consider throwing an error for the FE to catch so that we can let the user understand what might have gone wrong and what to do about it?

Reply:
Nice idea